docker-1_host/templates/resquev5/Dockerfile

```
@@ -17,12 +17,9 @@ COPY php.ini /etc/resque-serial/php.ini
 COPY supervisord.conf /etc/supervisord.conf
 COPY resque-5.conf /etc/supervisor.d/resque-5.conf
 COPY config-5.yml /etc/resque-serial/config-5.yml
-COPY resqu-server-5.2.1.zip /opt/qu/resqu-server-5.2.1.zip
+ADD https://raw.github.com/QualityUnit/resqu-server/5.4.0/build/php-resque-serial-5.4.0.zip /opt/qu
 
-RUN cd /opt/qu && unzip /opt/qu/resqu-server-5.2.1.zip && \
-    mv resqu-server-5.2.1/* /opt/qu/php-resque-5/ && \
-    rm -rf /opt/qu/resqu-server-5.2.1.zip && \
-    cd /opt/qu/php-resque-5 && curl -s http://getcomposer.org/installer | php && \
-    cd /opt/qu/php-resque-5 && php composer.phar install --no-dev --no-plugins --no-scripts
+RUN unzip /opt/qu/php-resque-serial-5.4.0.zip -d /opt/qu/php-resque-5/ && \
```

```suggestion
RUN curl -O https://raw.github.com/QualityUnit/resqu-server/5.4.0/build/php-resque-serial-5.4.0.zip && \
    echo "d0db7c4e544891dd4a8da949338a34f3d8bc23756f41e162e80f3b97e872a612  php-resque-serial-5.4.0.zip" | sha256sum -c && \
    unzip /opt/qu/php-resque-serial-5.4.0.zip -d /opt/qu/php-resque-5/ && \
```

This makes sure the archive is not left in the image even though it's unlinked later. Also, remote resources must be authenticated. Please, do the same thing everywhere else.

---

Suggested curl did not work. I believe that you meant to do something like this right - https://github.com/QualityUnit/LiveAgent-Docker/pull/6/commits/69f457453e6c57acb6d7cec7eff0d5db4a73983b ? If yes I will change it everywhere.

---

Let's do https://github.com/QualityUnit/LiveAgent-Docker/pull/6/files#r801751867 instead.